src/api/routes/meetings.ts

```diff
@@ -113,9 +114,49 @@ const decline = procedure
     商量解决方案。`, baseUrl);
 });
 
+/**
+ * List all meeting slots for admin management.
+ * Only MentorshipManager can access this.
+ */
+const listSlots = procedure
+  .use(authUser("MentorshipManager"))
+  .query(async () => {
+    return await db.MeetingSlot.findAll({
+      attributes: meetingSlotAttributes,
+      order: [["updatedAt", "DESC"]],
+    });
+  });
+
+/**
+ * Update a meeting slot's information.
+ * Only MentorshipManager can modify meeting slots.
+ */
+const updateSlot = procedure
+  .use(authUser("MentorshipManager"))
+  .input(
+    z.object({
+      id: z.number(),
+      meetingId: z.string().optional(),
+      meetingLink: z.string().url().optional(),
+    })
+  )
+  .mutation(async ({ input }) => {
+    const { id, ...updateData } = input;
+    
+    const slot = await db.MeetingSlot.findByPk(id);
+    if (!slot) throw notFoundError("数据", id.toString());
+    
+    return await db.MeetingSlot.update(updateData, {
```

Comment:
do not return anything. but

```
const updated = await db.MeetingSlot.update(...);
invariant(update <= 1);
if (update == 0) throw ...;
```

I assume update() returns the number of updated rows. but double check documentation to confirm.